mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -325,4 +325,21 @@ private[ml] trait HasStepSize extends Params {
   /** @group getParam */
   final def getStepSize: Double = $(stepSize)
 }
+
```

Comment:
`sharedParams.scala` can not be edited directly. Please look at `SharedParamsCodeGen.scala`.